Felienne/hedy
# Shows profile image in menubar

**Description**
In this PR we update the menubar to be better scale-able to new content, but more importantly... we introduce the profile image showing on the menubar! From now on the image chosen for the public profile will always be shown on the menubar, giving this a nice personal touch.


**Fixes**
This PR fixes #3204.

**How to test**
Make sure you are logged in and have a public profile. Your chosen image should be shown next to your username instead of the default icon. Verify that the icon is correctly updated when updating the public profile or when removing our profile.
